dnd5e/module/actor/entity.d.ts

```diff
@@ -116,7 +116,7 @@ declare class Actor5e extends Actor<Actor5e.Data, Item5e> {
    * @param itemData - The item data to use
    * @param options - Options which
    */
-  private _preCreateOwnedItem(itemData: Item5e.Data, options: any): void;
+  private _preCreateOwnedItem(itemData: Item5e['data'], options: any): void;
```

Comment:
I think, for consistency's sake that we should preserve Item5e.Data and Item5e.Data.Data (which will just mirror Item5e['data']). I'll push a commit to do this if you agree, it'd basically just be ```ts
type Data = Item5e['data'];
```

Reply:
This might have been complaining because `Item5e.Data` was generic